src/steps/01-AcquisitionPackageDetails/components/RequestingServiceAgency.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <ATATAutoComplete
+    :id="id"
+    :label="label"
+    :rules="rules"
+    class="_input-max-width mb-2"
+    :label-sr-only="false"
+    titleKey="text"
+    :searchFields="['text']"
+    :items="serviceOrAgencyData"
+    :selectedItem.sync="_selectedServiceOrAgency"
+    placeholder="Find your service/agency"
+    icon="arrow_drop_down"
+  />
+</template>

```

Comment:
This needs a "Preview" mode when `isForm` is false with just the label "Agency:" and the selected value - no dropdown - only text 
![image](https://user-images.githubusercontent.com/90333349/191993784-3c92efc1-488a-4527-8099-0ad908dad54d.png)